### ISSUE TITLE
Broken from ruby-asana gem update? 

### ISSUE THREAD

**Issue by emcniece** [goncaloborrega/TrelloToAsana#3]
```
$ ruby exportTrelloToAsana.rb 
exportTrelloToAsana.rb:43:in `<main>': undefined method `configure' for Asana:Module (NoMethodError)
```

Looks like the [Asana gem](https://github.com/Asana/ruby-asana) has been updated to have a slightly different syntax now.

**Comment by scrumpto** [comment]
Unfortunately I'm new to Ruby so even though I've programmed for years I can't seem to get this to work. I've worked on this for several days installing Ruby, updating Rubygems, trying to update the Trello and Asana gems, etc. but no go. Here's the output I'm receiving:

ruby exportTrelloToAsana.rb 
/Users/scrumpto/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `require': cannot load such file -- asana (LoadError)
    from /Users/scrumpto/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in`rescue in require'
    from /Users/scrumpto/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:39:in `require'
    from exportTrelloToAsana.rb:5:in`<main>'
